apps/explorer/src/hooks/useGetTransactionBlocks.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '@mysten/core';
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+import type { SuiAddress } from '@mysten/sui.js';
+
+export const DEFAULT_TRANSACTIONS_LIMIT = 20;
+
+// Fetch transaction blocks for an address, w/ toggle for to/from filter
+export function useGetTransactionBlocks(
```

Comment:
Probably should be `useGetTransactionBlocksForAddress`, or we need to generalize the filters more.